python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -2684,6 +2684,27 @@ def IntFloatModule_basic(module, tu: TestUtils):
     module.forward()
 
 
+# ==============================================================================
+
+class AtenSubFloatModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.value1 = 1.0
+        self.value2 = 2.0
+
+    @export
+    @annotate_args([
+        None,
+    ])
+    def forward(self):
+        return torch.ops.aten.sub.float(self.value1, self.value2)
```

Comment:
You need to remove the overload, i.e. do `torch.ops.aten.sub(...)`. See: https://github.com/llvm/torch-mlir/pull/1849#issuecomment-1428564242

Reply:
But I found that when I set `raw` output type, it is already folded：（maybe the folding happened in python level？）
<img width="1218" alt="image" src="https://user-images.githubusercontent.com/29956693/220981367-2bcd3529-1ac0-42fa-8c8e-948e1e3ff22c.png">

Reply:
If you use `self.value` instead of passing constants directly, does it work?

Reply:
Yes, you are right, thanks. 
Another problem, I found the E2E test failed locally:
![image](https://user-images.githubusercontent.com/29956693/221097857-39fe2429-2f44-4f9d-beeb-36aa6e9acfbc.png)

Reply:
You might need to add a cast `float(...)` to the result of `torch.ops.aten.sub` before returning. Some ops that return scalars will return a tensor when run on regular Python, so this extra cast is needed to make types match.

```python
In [2]: torch.ops.aten.sub(1.0, 1.0)
Out[2]: tensor(0.)
```

See for example:

https://github.com/llvm/torch-mlir/blob/3bed35c0ee5ed3cef4ccc00264051dea5b2983a3/python/torch_mlir_e2e_test/test_suite/scalar.py#L168-L173

Reply:
Thanks. BTW, there is something wrong in CI?

Reply:
Yeah, scalars were not properly handled by the torchdynamo backend. The following should fix it:

```diff
--- a/python/torch_mlir_e2e_test/configs/torchdynamo.py
+++ b/python/torch_mlir_e2e_test/configs/torchdynamo.py
@@ -59,10 +59,14 @@ def _refbackend_torchdynamo_backend(fx_graph: torch.fx.GraphModule,
     def compiled_callable(*inputs):
         inputs = [x.numpy() for x in inputs]
         result = loaded.forward(*inputs)
-        if not isinstance(result, tuple):
+        if isinstance(result, tuple):
+            result = tuple(torch.from_numpy(x) for x in result)
+        elif isinstance(result, torch.Tensor):
             result = torch.from_numpy(result)
+        elif isinstance(result, (bool, int, float)):
+            pass
         else:
-            result = tuple(torch.from_numpy(x) for x in result)
+            raise ValueError(f"Unhandled return type {type(result)}")
         return result
     return compiled_callable
```